packages/fetch-and-compile/lib/fetch.ts

```diff
@@ -94,6 +95,7 @@ async function tryFetchAndCompileAddress(
     } catch (error) {
       debug("error in getting sources! %o", error);
       failureReason = "fetch";
+      failureError = error;
```

Comment:
I might be missing something but It seems odd to introduce `failureError` and keep `failureReason`.

Reply:
That is good to know! I'll have to remember the different clients